dbt_common/__about__.py

```diff
@@ -0,0 +1 @@
+version = "0.0.1"
```

Comment:
Are we going with this for our first version? The recommendation from semver is to go with `0.1.0`, perhaps because you can't patch something that doesn't exist?

Source: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
I just went off my ticket.  I'll update to `0.1.0` per the recommendation.